File: bitget-java-sdk-api/src/main/java/com/bitget/openapi/api/v2/SpotAccountApi.java
package com.bitget.openapi.api.v2;

import com.bitget.openapi.dto.response.ResponseResult;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.POST;
import retrofit2.http.QueryMap;

import java.util.Map;

public interface SpotAccountApi {

    @GET("/api/v2/spot/account/info")
    Call<ResponseResult> info();

    @GET("/api/v2/spot/account/assets")
    Call<ResponseResult> assets(@QueryMap Map<String, String> paramMap);

    @GET("/api/v2/spot/account/bills")
    Call<ResponseResult> bills(@QueryMap Map<String, String> paramMap);

    @GET("/api/v2/spot/account/transferRecords")
    Call<ResponseResult> transferRecords(@QueryMap Map<String, String> paramMap);
}

File: bitget-php-sdk-api/src/api/v2/SpotAccountApi.php
<?php


namespace bitget\api\v2;


use bitget\internal\BitgetApiClient;

class SpotAccountApi
{
    var BitgetApiClient $BitgetApiClient;

    public function __construct($BitgetApiClient)
    {
        $this->BitgetApiClient = $BitgetApiClient;
    }

    public function info($params): string
    {
        return $this->BitgetApiClient->doGet("/api/v2/spot/account/info", $params);
    }

    public function assets($params): string
    {
        return $this->BitgetApiClient->doGet("/api/v2/spot/account/assets", $params);
    }

    public function bills($params): string
    {
        return $this->BitgetApiClient->doGet("/api/v2/spot/account/bills", $params);
    }

    public function transferRecords($params): string
    {
        return $this->BitgetApiClient->doGet("/api/v2/spot/account/transferRecords", $params);
    }

}